src/main/java/seedu/address/model/book/Book.java

```diff
@@ -67,12 +90,14 @@ public boolean equals(Object other) {
         }
 
         Book otherBook = (Book) other;
-        return otherBook.getTitle().equals(this.getTitle());
+        return otherBook.getIsbn().equals(this.getIsbn())
```

Comment:
Would it make sense to uniquely identify a book by its `gid` and/or `isbn` instead? Using `gid` might tie us to google books API though.